components/sync_manager/README.md

```diff
@@ -8,6 +28,10 @@ that `sync15` puts up.
 Unfortunately, places/logins/etc depend on sync15 directly, and so to prevent a
 dependency cycle (which is disallowed by cargo), doing so would require
 implementing the manager in a fully generic manner, with no specific handling of
-underlying crates. This seems extremely difficult to me, so this is split out
+underlying crates. This seems extremely difficult, so this is split out
 into it's own crate, which might happen to reach into the guts of `sync15` in
-some cases.
\ No newline at end of file
+some cases.
+
+Note also that the world has changed a little since then - in particular, we
+now have `sync15-traits` - there might now be an opportunity to move all the
+types into that crate, and merge `sync15` and `sync_manager`
```

Comment:
Thank you for adding these encouragements to our future selves that it's fine and good and worthwhile to pursue such cleanup work.